README.md

```diff
@@ -47,24 +52,122 @@ $ ember deploy
 
 ## ember-cli-deploy Hooks Implemented
 
-For detailed information on what plugin hooks are and how they work, please refer to the [Plugin Documentation][2].
+For detailed information on what plugin hooks are and how they work, please refer to the [Plugin Documentation][1].
 
 - `configure`
+- `setup`
+
+_Hooks that can be used for notifications:_
+
+- `willDeploy`
+- `willBuild`
+- `build`
+- `didBuild`
+- `willPrepare`
+- `prepare`
+- `didPrepare`
+- `willUpload`
+- `upload`
+- `didUpload`
+- `willActivate`
+- `activate`
 - `didActivate`
+- `teardown`
+- `fetchRevisions`
+- `displayRevisions`
+- `didFail`
 
 ## Configuration Options
 
-For detailed information on how configuration of plugins works, please refer to the [Plugin Documentation][2].
+For detailed information on how configuration of plugins works, please refer to the [Plugin Documentation][1].
 
 ###services
 
-An object that identifies all webhooks you want to notify of a successful deployment. You will put a key for every service you want to call after a successful deploy here.
+An object that identifies all webhooks you want to notify. You will put a key for every service you want to call on deploy here.
+
+A `service` configuration needs to provide four properties as configuration for
+`ember-cli-deploy-notifications` to know how to notify the service correctly:
+
+- `url` The url to call
+- `method` The HTTP-method to use for the call (defaults to `'POST'`)
+- `headers` A property to specify custom HTTP-headers (defaults to `{}`)
+- `body` The body of the request
+
+<hr/>
+**Whenever one of these properties returns a _falsy_ value, the service will _not_ be
+called.**
+<hr/>
```

Comment:
For people that follow this discussion: This hasn't been removed actually but is an implementation detail that you should not rely on when creating preconfigured services (i.e. it is discouraged to create preconfigured services that default to notify on any hook. This should be done by users directly in `config/deploy.js`)